example-mojaloop-backend/values.yaml

```diff
@@ -326,48 +197,25 @@ centralledger-obj:
   ## Enable persistence using Persistent Volume Claims
   ## ref: https://kubernetes.io/docs/user-guide/persistent-volumes/
   ##
-  persistence:
```

Comment:
Why was this removed?

Take note that persistence should always be disabled by default, otherwise default installation fail.

Reply:
same again